src/scripts/device_info.json

```diff
@@ -6870,6 +6870,31 @@
       ],
       "is_mockup_image_at_front": true,
       "available_perspectives": ["Portrait", "Landscape"]
+    },
+    {
+      "credits": "<p><a href=\"https://www.skymakers.digital/\" target=\"blank\">Skymaker's Devleoper</a></p>",
+      "color_str": "Default",
+      "color": null,
+      "meta_title": "Nokia 3310 Mock Up",
+      "meta_description": "1 click to generate your Nokia 3310 mockup! You can wrap screenshots in Nokia 3310 for prototypes.",
+      "display_resolution": [610, 433],
+      "device_type": "TV",
```

Comment:
Unbreakable? :rabbitsmile:

Reply:
maybe too much work if need to change design 🐶 let's do it when we free-er~